deploy/aws-lambda/README.md

```diff
@@ -0,0 +1,11 @@
+# Deploy to AWS Lambda
+
+You can deploy the webhook to AWS Lambda and activate it in your Kubernetes cluster by adding the following module to your Terraform project: 
+
+```terraform
+module "secrethub_mutating_webhook" {
+  source = "github.com/secrethub/secrethub-kubernetes-mutating-webhook?ref=v0.2.0/deploy/aws-lambda"
```

Comment:
Does this work? 🤔
First the `ref` query parameter and then the local path?

Reply:
I'd expect:
```suggestion
  source = "github.com/secrethub/secrethub-kubernetes-mutating-webhook//deploy/aws-lambda?ref=v0.2.0"
```
(didn't test this)

Reply:
Yes, indeed it works just like it is presented.
The one I tried was the following and it didn't work:
`github.com/secrethub/secrethub-kubernetes-mutating-webhook/deploy/aws-lambda?ref=v0.2.0`
Therefore I found the one presented.

Interesting enough, your suggestion works, so I think it is indeed cleaner. I will apply your version.

Reply:
Fixed.